src/Tasks/Microsoft.NET.Build.Tasks/GetNuGetShortFolderName.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using NuGet.Frameworks;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    public sealed class GetNuGetShortFolderName : TaskBase
+    {
+        [Required]
+        public string TargetFrameworkMoniker { get; set; }
+        
+        public string TargetPlatformMoniker { get; set; }
+
+
+        [Output]
+        public string NuGetShortFolderName { get; set; }
+
+        protected override void ExecuteCore()
+        {
+            NuGetShortFolderName = NuGetFramework
```

Comment:
@nkolev92 please double check this API use

Reply:
(this looks good to me?)

Reply:
Seems fine :) 
This could become interesting if anyone tries to call this task with UWP projects, but that's not an officially supported scenario ofc :)
Just keep it in mind in case you see some weirdness.